dependencies.props

```diff
@@ -8,7 +8,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
 
-    <PreReleaseLabel>preview1</PreReleaseLabel>
+    <PreReleaseLabel>b-uwp6</PreReleaseLabel>
```

Comment:
Just make this one change and undo the others.

Reply:
Will do.